src/Interface/InterfacesListContainer.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { isEqual, isEmpty, get } from 'lodash';
+import { FieldArray, getFormValues } from 'redux-form';
+
+import {
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+
+import InterfacesList from './InterfacesList';
+import { updateInterfaces } from './utils';
+
+class InterfacesListContainer extends Component {
+  static propTypes = {
+    parentMutator: PropTypes.object,
+    parentResources: PropTypes.object,
+    stripes: PropTypes.shape({
+      store: PropTypes.object
+    }),
+    orgId: PropTypes.string,
+  };
+
+  componentDidMount() {
+    const { parentMutator, parentResources } = this.props;
+    const interfaces = get(parentResources, 'vendorID.records[0].interfaces', []);
```

Comment:
wow... looks weird to me. All these things will be in `initialValues`, no? You can just get it from there and pass as a prop, can't you?

Reply:
Thank you, fixed.

Reply:
I'm not sure if you forgot to push the new code or have decided to leave as is

Reply:
I don't know why changes aren't presented here, but I pushed them.

Reply:
won't we have any problems with it after merge?

Reply:
Does it mean that there might be other changes what you've made, but they're not a part of the PR?